Mathlib/LinearAlgebra/Ray.lean

```diff
@@ -158,6 +161,26 @@ theorem pos_smul_left {r : R} (h : SameRay R x y) (hr : 0 < r) : SameRay R (r 
   h.nonneg_smul_left hr.le
 #align same_ray.pos_smul_left SameRay.pos_smul_left
 
+--TODO: Can we unify with `sameRay_nonneg_smul_right`?
+/-- A vector is in the same ray as a nonnegative integer multiple of itself. -/
+lemma _root_.sameRay_nsmul_right (v : M) (n : ℕ) : SameRay R v (n • v) := by
```

Comment:
```suggestion
lemma _root_.SameRay.nsmul_right (v : M) (n : ℕ) : SameRay R v (n • v) := by
```
for `(.nsmul_right _ _)` notation.

Reply:
Oh nevermind, I see this conflicts with below.

Reply:
I'd lean towards removing these, and forcing the user to write `refl.nsmul_right` or whatever.

Reply:
What about the similar ones for `smul` by `R` that already exist?

PS: Those conflicts, I think, are a pretty fundamental flaw of this new `(.bar _)` notation wrt our naming convention.